README.md

```diff
@@ -8,45 +8,39 @@
 | --------------- | ---------------------------------------------------------------- |
 | License:        | Apache License, Version 2.0                                      |
 | Build:          | NPM                                                              |
-| Issue tracker:  | https://issues.jboss.org/browse/AGCORDOVA                        |
-| Mailing lists:  | [aerogear-users](http://aerogear-users.1116366.n5.nabble.com/)   |
-|                 | [aerogear-dev](https://groups.google.com/forum/#!forum/aerogear) |
+| End User Documentation:  | https://docs.aerogear.org - Under construction - [Github Link](./docs/modules/ROOT/pages/)|
+| Community Documentation: | https://aerogear.org             |
+| Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR                        |
+| Mailing lists:  | [aerogear-dev](https://groups.google.com/forum/#!forum/aerogear) |
 
 AeroGear SDK for JavaScript, Apache Cordova and React Native.
 
-## Documentation
+## Showcase application
 
-1. [End User Getting Started Guide](./docs/getting-started.adoc)
+To try out the latest release of the SDK go to our [Cordova Showcase Template Application](https://github.com/aerogear/cordova-showcase-template)
 
+### List of Modules
 
-### List of SDKs
+This Aerogear SDK consists of a set of Modules.
 
-AeroGear Services SDK consist of set of separate SDKs
-
-- [Auth](./docs/auth/README.adoc): Mobile Auth SDK
-- [Metrics](./docs/metrics/metrics.adoc): Mobile application metrics SDK
-- [Security](./docs/security/security.adoc): Mobile Security Checks SDK
+- [Auth](./docs/modules/ROOT/pages/auth.adoc): Mobile Application Auth Module
```

Comment:
@ciaranRoche for tho other readme's we removed these links althogether. Also see [this discussion ](https://groups.google.com/forum/#!topic/aerogear/pdP8BFfxcTw)about how the docs should be named.

Reply:
Good catch, I will update those doc names.

@laurafitzgerald I do think we should provide links to these docs from the ReadMe, I don't think many people would navigate through the folder structure to get to them. What do you think?